webextensions/manifest/chrome_url_overrides.json

```
@@ -95,7 +95,8 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "14.1",
+                "notes": "The user may select a specific extension to override the new tab or window page in Safari General Preferences."
```

If I understand this note, then developers can set `newtab`, but it won't take effect unless the user chooses the extension in Safari's preferences. If that's correct, we should make that relationship more explicit in the note. Something like:

```suggestion
                "notes": "An extension can define a custom new tab or window page, but it does not take effect until the user chooses the extension to override the page in Safari's <em>General</em> preferences."
```

(I based the formatting of the general preferences text based on my machine, but I'm not running the latest version. If _Safari General Preferences_ is the name of the window or tab, then please revise that bit)

---

Addressed in 2037915.